### ISSUE TITLE
Return code incorrect when piping to bcftools sort from failing bcftools concat

### ISSUE THREAD

**Issue by wdecoster** [samtools/bcftools#898]
I have something which looks like a bug, using bcftools v1.7:

```
$ bcftools concat -a file1.vcf.gz file2.vcf.gz | bcftools sort - -o out.vcf
Writing to /tmp/bcftools-sort.AqH5mj
[W::bcf_hdr_merge] Trying to combine "SVLEN" tag definitions of different lengths
[W::vcf_parse] INFO 'SEQ' is not defined in the header, assuming Type=String
[E::bcf_translate] Unchecked error (2), exiting
Merging 0 temporary files
Cleaning
Done
```

bcftools concat fails on `INFO 'SEQ' is not defined...`, leading to the unchecked error. That's fine, the header is incomplete. I can fix that. I wouldn't be angry if `bcftools` would have a `--dude-trust-me` option, but that's not the main issue here. 

But the bcftools sort happily sorts the empty input, and the entire process returns a return code 0:

```
$ echo $?
0
```

That's not okay I guess, and this causes downstream issues in my snakemake workflow.

Or is this something I can solve?

**Comment by pd3** [comment]
I believe this should be fixed already by 69711c976. Can you please try with the most recent version of bcftools?

**Comment by pd3** [comment]
I will assume the problem was fixed and close the issue now. If not, please let me know.
